src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCountTableColumns.java

```diff
@@ -6,7 +6,8 @@
  * Created by davidben on 11/30/15.
  */
 public enum AllelicCountTableColumns {
-    CONTIG("CONTIG"), POSITION("POS"), REF_COUNT("REF_COUNT"), ALT_COUNT("ALT_COUNT");
+    CONTIG("CONTIG"), POSITION("POS"), REF_COUNT("REF_COUNT"), ALT_COUNT("ALT_COUNT"), REF_NUCLEOTIDE("REF_NUCLEOTIDE"),
```

Comment:
I had kept it this way for backwards compatibility but I don't think we are going to have a problem (the het pulldown is an intermediate file and we don't have a release with CNV tools yet).